src/app/workflow/workflow.component.ts

```diff
@@ -206,11 +207,6 @@ export class WorkflowComponent extends Entry {
         this.setUpWorkflow(workflow);
       }
     );
-    this.workflowService.copyBtn$.pipe(takeUntil(this.ngUnsubscribe)).subscribe(
```

Comment:
Just making sure that you intended to remove this and line 63? I can't tell what it was doing, which is probably an indication that it should be removed, but just checking.

Reply:
i search the project for that workflow copy button and nothing seems to use it, so I removed it.